json/schema/fair_genomic_tracks.schema.json

```
@@ -0,0 +1,420 @@
+{
+    "$schema": "http://json-schema.org/draft-06/hyper-schema#",
```

I've faced a problem with this line when tested this schema in TrackFind. I'm using [this library](https://github.com/everit-org/json-schema) to deal with JSON schema and its documentation says:
> - if there is "$schema": "http://json-schema.org/draft-04/schema" in the schema root, then Draft 4 will be used
> - if there is "$schema": "http://json-schema.org/draft-06/schema" in the schema root, then Draft 6 will be used
> - if there is "$schema": "http://json-schema.org/draft-07/schema" in the schema root, then Draft 7 will be used
> - if none of these is found then Draft 4 will be assumed as default

So if something custom is placed there (like `hyper-schema`) this library throws an exception. I'm not sure what's the expected behavior and if this is correct, but we need to deal with it somehow.

---

As far as I can understand, we do not need hyper-schema. JSON hyper-schema seems to be a way to design more advanced APIs where for instance the returned JSON contains links to how the content can be used in the API. A bit like the move from structs to objects, if I understand correctly. Seems very useful, but perhaps not that much used yet? Here is an example that is relatively easy to understand (most other explanations were very hard to read for a JSON nwebie): http://benfalk.com/blog/2016/02/09/json-hyper-schema/

I see no reason why we should have a hyper-schema, and suggest we change into a regular schema. It is probably a mistake.
```suggestion
    "$schema": "http://json-schema.org/draft-06/schema#",
```